app/views/browse/show.html.erb

```diff
@@ -22,6 +22,11 @@
   <h1 class="browse__heading govuk-heading-xl">
     <%= page.title %>
   </h1>
+  <%= render "govuk_publishing_components/components/lead_paragraph", {
```

Comment:
This is currently adding a lead second paragraph to the browse pages. Making both the CoL and browse pages use the same header partial feels like a sensible change but one we could do when we do the refactoring later. So it might be safer to delete the changes to this file for now to avoid changing the live pages.

Reply:
oh i forgot to delete the p element below it, should i do that or should i remove this lead paragraph component